src/obb.cpp

```diff
@@ -0,0 +1,206 @@
+#include <geometric_shapes/obb.h>
+
+#include <fcl/config.h>
+
+#if FCL_MAJOR_VERSION == 0 && FCL_MINOR_VERSION == 5
```

Comment:
Remove FCL 0.5 check.

Reply:
To confirm, the FCL 0.5 checks are to be removed, the code inside the if check is kept, and the code inside the else block is removed for all 0.5 checks?